.github/workflows/pr-test.yml

```diff
@@ -153,7 +153,8 @@ jobs:
 
           ARGS=()
           for file in "${files_to_build[@]}"; do
-              ARGS+=("-f" "$PWD/$file")
+              file_without_prefix="${file#files/}"
```

Comment:
PS: I checked locally that this removes `files/` only once (given that there are some pages about `files` attributes that include `/files/` twice).